README.md

```diff
@@ -203,17 +240,21 @@ If the SSH or SCP command fails, ensure that the `dp remote allow` command has b
 
 #### Remote Allow security group error
 
-`Error: no security groups matching environment: "develop" with name "develop - bastion"`
+`Error: no security groups matching environment: "sandbox" with name "sandbox - bastion"`
 
-Ensure you have `region=eu-west-1` in your AWS configuration.
+Ensure you have `region=eu-west-2` in your AWS configuration.
 
-Depending on the command you're trying to run, and what you're trying to access, ensure your `AWS_DEFAULT_PROFILE` is set correctly.
+Depending on the command you're trying to run, and what you're trying to access, ensure your `AWS_PROFILE` is set correctly.
+Example:
+```yaml
+export AWS_PROFILE=dp-staging
+```
```

Comment:
We should also mention here to ensure that there is no aws-b config in the `~/.aws/credentials` file.